cinn/common/ir_util.cc

```diff
@@ -49,8 +50,17 @@ Expr RampRelatedAdd(ir::Broadcast *broadcast, Expr other) {
   CHECK_EQ(other.type().lanes(), 1);
   return ir::Broadcast::Make(broadcast->value + other, broadcast->lanes);
 }
-// ramp * ramp
+// ramp + ramp
 Expr RampRelatedAdd(ir::Ramp *ramp, ir::Ramp *other) {
+  CHECK(ramp);
+  CHECK(other);
+  if (ramp->lanes == other->lanes) {
+    Expr base_add   = common::AutoSimplify(ramp->base + other->base);
+    Expr stride_add = common::AutoSimplify(ramp->stride + other->stride);
+    LOG(INFO) << base_add;
```

Comment:
Change LOG to VLOG in module code.

LOG is only allowed in tests.

Reply:
done,thanks~